src/EA.Weee.Web.Tests.Unit/Areas/Admin/ViewModels/UploadObligationsViewModelMapTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace EA.Weee.Web.Tests.Unit.Areas.Admin.ViewModels
+{
+    using System;
+    using AutoFixture;
+    using Core.Shared;
+    using FluentAssertions;
+    using Web.Areas.Admin.Mappings.ToViewModel;
+    using Xunit;
+
+    public class UploadObligationsViewModelMapTests
```

Comment:
This has the same name as another test class? Is it just a quirk of naming convention or an accidental copy paste